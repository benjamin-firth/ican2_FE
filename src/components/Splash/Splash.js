import React from 'react';
import './Splash.scss';
import { Link } from 'react-router-dom';

const Splash = () => {

  return (
    <section className='splash-container'>
      <p>welcome! <br /> we are so glad you're here.</p>
      <button className='login-button'>log in</button>
      <button className='sign-up-button'>sign up</button>
      <Link to='/about'>learn more</Link>
    </section>
  );
}

export default Splash;
