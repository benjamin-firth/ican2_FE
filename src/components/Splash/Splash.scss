@import '../../variables.scss';

.splash-container {
  display: flex;
  flex-direction: column;
  align-items: center;
  background-color: $primarycolor;
  height: 100vh;
  padding-top: 5vw;
  p {
    font-family: $secondaryfont;
    color: $secondarycolor;
    font-size: 1.3vw;
  }
  button {
    width: 15vw;
    margin: .5vw;
    padding: 1vw;
    border: 5px solid $secondarycolor;
    color: $secondarycolor;
    font-size: 1.5vw;
    font-family: $secondaryfont;
    &:hover {
      background-color: $secondarycolor;
      color: white;
    }
  }
}
