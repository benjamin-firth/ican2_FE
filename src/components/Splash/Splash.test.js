import React from 'react';
import { shallow } from 'enzyme';
import Splash from './Splash';

describe('Splash', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow( <Splash />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});