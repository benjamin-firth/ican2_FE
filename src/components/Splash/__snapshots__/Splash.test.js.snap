// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash should match the snapshot 1`] = `
<section>
  <p>
    This the splash page.
  </p>
</section>
`;
